bench-tps/src/cli.rs

```diff
@@ -496,22 +497,133 @@ pub fn extract_args(matches: &ArgMatches) -> Config {
             .value_of("instruction_padding_program_id")
             .map(|target_str| target_str.parse().unwrap())
             .unwrap_or_else(|| FromOtherSolana::from(spl_instruction_padding::ID));
+        let parsed_data_size = data_size
+            .to_string()
+            .parse()
+            .map_err(|_| "Can't parse padded instruction data size")?;
         args.instruction_padding_config = Some(InstructionPaddingConfig {
             program_id,
-            data_size: data_size
-                .to_string()
-                .parse()
-                .expect("Can't parse padded instruction data size"),
+            data_size: parsed_data_size,
```

Comment:
**&lt;minor-style&gt;**

I think it is totaly fine to shadow `data_size` here:

```rust
        let data_size = data_size
            .to_string()
            .parse()
            .map_err(|_| "Can't parse padded instruction data size")?;
        args.instruction_padding_config = Some(InstructionPaddingConfig {
            program_id,
            data_size,
        });
```

**&lt;/minor-style&gt;**

Reply:
By the way, `.to_string()` is unnecessary here.
`.parse()` works just fine on a `&str`.

Reply:
done